ludwig/utils/loss_utils.py

```diff
@@ -3,7 +3,10 @@
 
 def rmspe_loss(targets: torch.Tensor, predictions: torch.Tensor) -> torch.Tensor:
     """Root mean square percentage error."""
```

Comment:
```suggestion
    """
    Root mean square percentage error.
    
    Bad predictions can lead to arbitrarily large RMSPE values, 
    especially if some values of targets are very close to zero. 
    We return a large value instead of inf when (some) targets are zero.
    """
```